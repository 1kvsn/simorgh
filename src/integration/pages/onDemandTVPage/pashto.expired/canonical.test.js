/**
 * @service pashto
 * @pathname /pashto/bbc_pashto_tv/tv/w172xcldhhrhmcf
 */

import runExpiredEpisodeTests from '../expiredEpisodeTests';

describe('Canonical Pashto On Demand TV Page', () => {
  runExpiredEpisodeTests();
});
