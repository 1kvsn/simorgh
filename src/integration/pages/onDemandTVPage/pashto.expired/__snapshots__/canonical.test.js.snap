// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Pashto On Demand TV Page A11y I can see the skip to content link 1`] = `"مطلب ته ورشئ"`;

exports[`Canonical Pashto On Demand TV Page Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=68&amp;p=pashto.bbc_pashto_tv.tv.w172xcldhhrhmcf.page&amp;x1=[urn:bbc:pips:w172xcldhhrhmcf]&amp;x2=[responsive]&amp;x3=[news-pashto]&amp;x4=[ps]&amp;x7=[player-episode]&amp;x8=[simorgh-nojs]&amp;x9=[نړۍ+دا+وخت+-+BBC+News+پښتو]\\"/>"`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the 'Content is not available' placeholder 1`] = `"دغه فایل نور د لاسرسي وړ نه دی."`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the brand title 1`] = `"نړۍ دا وخت"`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the episode summary 1`] = `"د بي بي سي پښتو ټلویزیوني خپرونه چې هره ورځ د افغانستان په شپږ بجو په ژوندۍ بڼه خپرېږي. دلته یې لیدلی شئ."`;

exports[`Canonical Pashto On Demand TV Page Expired Episode I can see the episode title 1`] = `"۲۸ می ۲۰۲۰"`;

exports[`Canonical Pashto On Demand TV Page Expired Episode a11y Assistive technology reads the brand and episode title as the headline 1`] = `"نړۍ دا وخت, ۲۸ می ۲۰۲۰"`;

exports[`Canonical Pashto On Demand TV Page Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pashto"`;

exports[`Canonical Pashto On Demand TV Page Footer I can see the footer copyright and external linking text 1`] = `"© 2020 بي بي سي. بي بي‌ سي‌ د نورو ویبپاڼو د محتوا مسوله نه ده. د نورو ویبپاڼو لینکولو په اړه زموږ تګلاره."`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 1`] = `
Object {
  "text": "BBC News, پښتو",
  "url": "/pashto",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 2`] = `
Object {
  "text": "ولې تاسې پر بي بي سي خبرونو باور کولی شئ",
  "url": "https://www.bbc.com/pashto/institutional-49283007",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 3`] = `
Object {
  "text": "د کارولو شرايط",
  "url": "https://www.bbc.com/pashto/institutional-37620701",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 4`] = `
Object {
  "text": "د بي بي سي‌ په اړه",
  "url": "https://www.bbc.com/pashto/institutional-37622761",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 5`] = `
Object {
  "text": "د محرميت تگلاره",
  "url": "https://www.bbc.com/pashto/institutional-37622762",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 6`] = `
Object {
  "text": "Cookies",
  "url": "https://www.bbc.com/usingthebbc/cookies/",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 7`] = `
Object {
  "text": "زموږ سره اړیکي",
  "url": "https://www.bbc.co.uk/pashto/institutional-37622768",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 8`] = `
Object {
  "text": "AdChoices / Do Not Sell My Info",
  "url": "https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/",
}
`;

exports[`Canonical Pashto On Demand TV Page Footer Link should match text and url 9`] = `
Object {
  "text": "د نورو ویبپاڼو لینکولو په اړه زموږ تګلاره.",
  "url": "https://www.bbc.co.uk/help/web/links/",
}
`;

exports[`Canonical Pashto On Demand TV Page Header I can see the branding 1`] = `"BBC News, پښتو"`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 1`] = `
Object {
  "text": "کور پاڼه",
  "url": "/pashto",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 2`] = `
Object {
  "text": "افغانستان",
  "url": "/pashto/afghanistan",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 3`] = `
Object {
  "text": "پښتونخوا",
  "url": "/pashto/topics/cz74k7wy49jt",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 4`] = `
Object {
  "text": "نړۍ",
  "url": "/pashto/world",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 5`] = `
Object {
  "text": "ښځې",
  "url": "/pashto/topics/e45cb5f8-3c87-4ebd-ac1c-058e9be22862",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 6`] = `
Object {
  "text": "لوبې",
  "url": "/pashto/sport",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 7`] = `
Object {
  "text": "ويډيوګانې",
  "url": "/pashto/media/video",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 8`] = `
Object {
  "text": "ځانګړې پاڼې",
  "url": "/pashto/in_depth",
}
`;

exports[`Canonical Pashto On Demand TV Page Header Navigation link should match text and url 9`] = `
Object {
  "text": "کالم",
  "url": "/pashto/columns",
}
`;

exports[`Canonical Pashto On Demand TV Page I can see the brand title 1`] = `"نړۍ دا وخت"`;

exports[`Canonical Pashto On Demand TV Page I can see the episode summary 1`] = `"د بي بي سي پښتو ټلویزیوني خپرونه چې هره ورځ د افغانستان په شپږ بجو په ژوندۍ بڼه خپرېږي. دلته یې لیدلی شئ."`;

exports[`Canonical Pashto On Demand TV Page I can see the episode title 1`] = `"۲۸ می ۲۰۲۰"`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 1`] = `
Object {
  "sizes": null,
  "url": "http://localhost:7080/pashto/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 2`] = `
Object {
  "sizes": "72x72",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-72x72.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 3`] = `
Object {
  "sizes": "96x96",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-96x96.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 4`] = `
Object {
  "sizes": "128x128",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-128x128.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 5`] = `
Object {
  "sizes": "144x144",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-144x144.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 6`] = `
Object {
  "sizes": "152x152",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-152x152.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 7`] = `
Object {
  "sizes": "180x180",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-180x180.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 8`] = `
Object {
  "sizes": "192x192",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 9`] = `
Object {
  "sizes": "384x384",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-384x384.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Apple Touch Icon should match attributes 10`] = `
Object {
  "sizes": "512x512",
  "url": "https://news.files.bbci.co.uk/include/articles/public/pashto/images/icons/icon-512x512.png",
}
`;

exports[`Canonical Pashto On Demand TV Page SEO Canonical link 1`] = `"http://localhost:7080/pashto/bbc_pashto_tv/tv/w172xcldhhrhmcf"`;

exports[`Canonical Pashto On Demand TV Page SEO Dir attribute 1`] = `"rtl"`;

exports[`Canonical Pashto On Demand TV Page SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical Pashto On Demand TV Page SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical Pashto On Demand TV Page SEO Heading level 1 1`] = `"نړۍ دا وخت, ۲۸ می ۲۰۲۰"`;

exports[`Canonical Pashto On Demand TV Page SEO Lang attribute 1`] = `"ps"`;

exports[`Canonical Pashto On Demand TV Page SEO Linked data should match text 1`] = `
Object {
  "@context": "http://schema.org",
  "@graph": Array [
    Object {
      "@type": "WebPage",
      "image": Object {
        "@type": "ImageObject",
        "height": 576,
        "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png",
        "width": 1024,
      },
      "mainEntityOfPage": Object {
        "@id": "http://localhost:7080/pashto/bbc_pashto_tv/tv/w172xcldhhrhmcf",
        "@type": "WebPage",
        "name": "نړۍ دا وخت",
      },
      "publisher": Object {
        "@type": "NewsMediaOrganization",
        "logo": Object {
          "@type": "ImageObject",
          "height": 576,
          "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png",
          "width": 1024,
        },
        "name": "BBC News پښتو",
        "publishingPrinciples": "https://www.bbc.com/pashto/institutional-49283007",
      },
      "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png",
      "url": "http://localhost:7080/pashto/bbc_pashto_tv/tv/w172xcldhhrhmcf",
    },
  ],
}
`;

exports[`Canonical Pashto On Demand TV Page SEO OG description 1`] = `"د بي بي سي پښتو ټلویزیوني خپرونه چې هره ورځ د افغانستان په شپږ بجو په ژوندۍ بڼه خپرېږي. دلته یې لیدلی شئ."`;

exports[`Canonical Pashto On Demand TV Page SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png"`;

exports[`Canonical Pashto On Demand TV Page SEO OG image alt 1`] = `"BBC News پښتو"`;

exports[`Canonical Pashto On Demand TV Page SEO OG locale 1`] = `"ps"`;

exports[`Canonical Pashto On Demand TV Page SEO OG site name 1`] = `"BBC News پښتو"`;

exports[`Canonical Pashto On Demand TV Page SEO OG title 1`] = `"نړۍ دا وخت - BBC News پښتو"`;

exports[`Canonical Pashto On Demand TV Page SEO OG type 1`] = `"website"`;

exports[`Canonical Pashto On Demand TV Page SEO OG url 1`] = `"http://localhost:7080/pashto/bbc_pashto_tv/tv/w172xcldhhrhmcf"`;

exports[`Canonical Pashto On Demand TV Page SEO Page title 1`] = `"نړۍ دا وخت - BBC News پښتو"`;

exports[`Canonical Pashto On Demand TV Page SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical Pashto On Demand TV Page SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical Pashto On Demand TV Page SEO Twitter creator 1`] = `"@bbcpashto"`;

exports[`Canonical Pashto On Demand TV Page SEO Twitter description 1`] = `"د بي بي سي پښتو ټلویزیوني خپرونه چې هره ورځ د افغانستان په شپږ بجو په ژوندۍ بڼه خپرېږي. دلته یې لیدلی شئ."`;

exports[`Canonical Pashto On Demand TV Page SEO Twitter image alt 1`] = `"BBC News پښتو"`;

exports[`Canonical Pashto On Demand TV Page SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png"`;

exports[`Canonical Pashto On Demand TV Page SEO Twitter site 1`] = `"@bbcpashto"`;

exports[`Canonical Pashto On Demand TV Page SEO Twitter title 1`] = `"نړۍ دا وخت - BBC News پښتو"`;

exports[`Canonical Pashto On Demand TV Page a11y Assistive technology reads the brand and episode title as the headline 1`] = `"نړۍ دا وخت, ۲۸ می ۲۰۲۰"`;
